app/controllers/plans_controller.rb

```diff
@@ -23,9 +23,9 @@ def new
     # Get all of the available funders and non-funder orgs
     @funders = Org.funder
                   .joins(:templates)
-                  .where(templates: { published: true }).uniq.sort(&:name)
+                  .where(templates: { published: true }).uniq.sort_by(&:name)
```

Comment:
How much difference is there between sort and sort_by ?

Reply:
Depends how academic you want to get about it. There are performance differences, since `sort_by` uses more effective algorithm behind the scenes. But the main practical difference is that `sort` doesn't take a method name proc in the way I used it above. So this was a bug.